packages/trader/src/Assets/Trading/Categories/trade-categories-gif.jsx

```diff
@@ -16,11 +16,11 @@ import ImageRunHighLow from 'Assets/SvgComponents/trade_explanations/img-run-hig
 import ImageSpread from 'Assets/SvgComponents/trade_explanations/img-spread.svg';
 import ImageTickHighLow from 'Assets/SvgComponents/trade_explanations/img-tick-high-low.svg';
 import ImageTouch from 'Assets/SvgComponents/trade_explanations/img-touch.svg';
-import ImageVanilla from 'Assets/SvgComponents/trade_explanations/img-vanilla.svg';
 import ContractTypeDescriptionVideo from './contract-type-description-video';
 
 // TODO: Replace static image svgs with themed GIFs or animated SVGs
 const TradeCategoriesGIF = ({ category, selected_contract_type }) => {
+    if (category !== selected_contract_type) return null;
```

Comment:
This checked was added to prevent situations when a hidden video started to play